server/views/pages/bookedtoday/arrivals/autoMatchingRecords/multipleMatchingRecordsFound.njk

```diff
@@ -65,6 +65,14 @@
                             </fieldset>
                         </div>
                 </div> 
+                <details class="govuk-details govuk-!-margin-bottom-4">
+                    <summary class="govuk-details__summary">
+                        <span class="govuk-details__summary-text" id="incorrect_vehicle_details_reveal" data-qa="create-new-details-reveal">If you cannot find the correct record</span>
```

Comment:
vehicle?

Reply:
`incorrect_vehicle_details_reveal` copied over from the same component used on the other (unexpected) `multipleExistingRecordsFound.nkj` file, could I rename toe ids across each multiple match page file to `create_new_details_reveal`? If that makes more sense.

Reply:
Renamed to `create_new_details_reveal`